include/pqxx/largeobject.hxx

```diff
@@ -38,33 +36,33 @@ public:
 
   /// Refer to a nonexistent large object (similar to what a null pointer
   /// does).
-  largeobject() noexcept = default;
+  [[deprecated]] largeobject() noexcept = default;
```

Comment:
These can take a message. I normally like to put in a pointer to what is replacing it

Reply:
Good point, thanks.  I've added a message.